DataFormats/TrackReco/interface/DeDxHitInfo.h

```diff
@@ -32,8 +34,10 @@ namespace reco {
       DetId detId_;
       //! hit position
       LocalPoint pos_;
+      uint8_t type_;
     };
 
+    enum HitType { Complete = 0, Compatible = 1, Calibration = 2 };
```

Comment:
Same here: avoid comparisons and math operations between enum's and integer's

Reply:
In this case the enum are only used in "if" comparisons. So in which circumstances would it be fine to use enums?

Reply:
in Indexing

Reply:
Addressed in 05c2dc85edef1e8e3a387cd8694941aa4c4e24bd